CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ include (buildutils/OpenMPInstaller.cmake)
 
 set (CMAKE_CXX_STANDARD 11)
 
+set (CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
It suppose it could be, but I'm not sure what the advantages are.  

Writing a file with compile-commands seems to be a fast operation, and it's something I wished I had easier access to in the past.  Are there reasons I'm missing that we should avoid it?